end-to-end-tests/specs/screenshot.spec.js

```diff
@@ -0,0 +1,102 @@
+var assert = require('assert');
+var expect = require('chai').expect;
+
+const CBIOPORTAL_URL = process.env.CBIOPORTAL_URL.replace(/\/$/, "");
+
+function ssAssert(result, message){
+    assert(result[0].isWithinMisMatchTolerance);
+};
+
+describe('fdas', function(){
```

Comment:
fdas?

Reply:
yeah, i just needed to run these on travis to see how it handled the screenshots.  will fix this up.